Created Install Script for Debian (#3041)
AleoHQ/snarkOS: README.md

cd snarkOS
```

Lastly, install `snarkOS`:
```
cargo install --locked --path .
```

> Please ensure ports `4133/tcp` and `3033/tcp` are open on your router and OS firewall.

Reviewer: Let's get rid of port recommendations. Also these are not even the defaults? 🤔

./build_ubuntu.sh
```

> Please ensure ports `4133/tcp` and `3033/tcp` are open on your router and OS firewall.

Reviewer 2: why don't we move this right after the `cd snarkOS` step, so we don't duplicate it for the scripts?

Author: done

Please ensure ports `4133/tcp` and `3033/tcp` are open on your router and OS firewall.

> **[*Beta* - For Debian users]** 

Reviewer 2: ```suggestion
**[For Debian users]** 
```

> Firstly, install the updated version of LLVM and Clang.

Reviewer 2: ```suggestion
Firstly, install the most recent versions of LLVM and Clang.
```

Download the official install script from here (https://apt.llvm.org/llvm.sh) and then run:
```
./llvm.sh
```

A helper script to install dependencies is available. From the `snarkOS` directory, run:
```
./build_debian.sh
```

> Lastly, install `snarkOS`:
> ```
> cargo install --locked --path .
> ```
> 

Reviewer 2: ```suggestion
```
This is already contained in the script, so no need for it; can you also remove it in the Ubuntu docs right above?

Author: I removed those lines from ubuntu and debian instructions.
